CMakeLists.txt

```
@@ -172,6 +173,7 @@ install(FILES
   config/franka_control_node.yaml
   config/pose_publisher_params.yaml
   config/start_joint_pose.yaml
+  config/obstacle_infos.yaml
```

```suggestion
  config/obstacle_params.yaml
```
This is more suitable name in the ROS spirit

---

I am bumping this comment as it was still not changed